src/public/curriculum/stretching-and-shrinking/stretching-and-shrinking.json

```diff
@@ -36,9 +36,7 @@
       "guess-and-check": "Guess and Check",
       "none": "None"
     },
-    "tagPrompt": {
-      "default": "Select Student Strategy"
-    }
```

Comment:
@scytacki @kswenson @tealefristoe, in the last PR https://github.com/concord-consortium/collaborative-learning/pull/1803

This data structure should've been just a string and not an object, could we quickly get this merged into master ?